### ISSUE TITLE
Auto Hide Menu Bar does not reveal on trigger, Win 8.1

### ISSUE THREAD

**Issue by polarathene** [atom/atom#7806]
Atom 1.0.0, Win 8.1 x64

When the setting is checked/toggled on, I can only trigger reveal of the menu bar via `alt` on a tab such as 'Settings', on any of my code tabs pressing `alt` does nothing. Is it also intended to hide when focus changes rather than waiting on `alt` key again?

If there is a way for me to backup/snapshot my current installation of Atom with packages/settings that I can later restore, I could try verify the issue still exists on a fresh install. My monitor also has 166dpi, I've noticed the menu bar lacks vertical whitespace, it's packed rather tightly against the text, if there is a way to add some vertical padding or adjust the menu bars height that'd be good too :)

**Comment by peol** [comment]
I experience similar, but found that it _does_ work, if you give something other than the code area focus (like clicking in the tab bar next to a tab) first.

**Comment by krilim** [comment]
I use linux and  experience the same... When I have the menu hidden it does not reveal itself via alt...

**Comment by polarathene** [comment]
@peol I can confirm clicking in other areas like the blank space of the tab bar(if available) or in the treeview will trigger reveal via alt. Is the code pane focus highjacking `alt` input for another keymap with higher priority perhaps? I like hiding the menu bar with dark theme but it is not practical if I want to reveal it at a later time with current situation

**Comment by kwinne** [comment]
Same problem here, I'm using windows 7 and alt key is not working to show the menu if I'm in the code area. It does work when I click on tree.

**Comment by afontaine** [comment]
On my install, it's clashing with a keybinding set by [the linter package](https://github.com/atom-community/linter). See atom-community/linter#648 for more info.

**Comment by mnquintana** [comment]
@polarathene Can you reproduce this in safe mode? (`atom --safe`)

**Comment by mikeott** [comment]
@afontaine That works for me. Thanks.

**Comment by polarathene** [comment]
@mnquintana `atom --safe` does not have the issue, `alt` reveals menu on code tabs. I also tried @afontaine 's advice and disabled linter on atom without `--safe` and this also let `alt` reveal the menu on code tabs.

**Comment by mnquintana** [comment]
Thanks for the update! This is a duplicate of https://github.com/atom-community/linter/issues/648 thrn.
